src/patternfly/components/Content/content.scss

```diff
@@ -63,16 +63,15 @@
   --pf-c-content--a--hover--TextDecoration: var(--pf-global--link--TextDecoration--hover);
 
```

Comment:
`--pf-c-content--h1--MarginTop: var(--pf-global--spacer--lg);` applies to h1 through to h6, should we combine them as one variable?

Reply:
This is a good question. It's also a fair question for the margin-bottom, line-height, and font-weight properties since there are 2 values we use for that across all the headings. I'll ask @mceledonia how he feels about creating shared vars for this and the title component.

Reply:
Chatted with mr @mceledonia and he's down with combining the shared values between these vars so that the consistent values between them should remain consistent if one of them changes.